interface/main/calendar/modules/PostCalendar/pntemplates/default/views/week/ajax_template.html

```diff
@@ -355,9 +355,8 @@
   foreach ($providers as $provider)
    if ($provider['username'] == $username) echo " selected";
   echo ">" . text($doc['lname']) . ", " . text($doc['fname']) . "</option>\n";
- }
  echo "   </select>\n";
-
+}
```

Comment:
hi @juggernautsei , looks like you have a residual change here that is unintended?